internal/sessions/state.go

```diff
@@ -116,18 +116,11 @@ func (s State) RouteSession() *State {
 
 // Verify returns an error if the users's session state is not valid.
 func (s *State) Verify(audience string) error {
-	if s.NotBefore != nil && timeNow().Before(s.NotBefore.Time()) {
```

Comment:
These will still be checked on every request as part of AuthZ. 

Functionally, authN really only uses  Verify now to to check if the session is expired, and therefore whether or not to try and refresh the session.